cmd/flow/main.go

```diff
@@ -117,8 +119,13 @@ func main() {
 	cmd.AddCommand(snapshot.Cmd)
 	cmd.AddCommand(super.FlixCmd)
 	cmd.AddCommand(super.GenerateCommand)
+	cmd.AddCommand(dependencymanager.Cmd)
 
 	command.InitFlags(cmd)
+	cmd.AddGroup(&cobra.Group{
```

Comment:
Better to make sure the super commands are always on top. Contract manager should be after